doc/conf.py

```diff
@@ -7,6 +7,4 @@
 
 
 _g = globals()
-_g.update(build_package_configs(
-    project_name='ts_sal',
-    version='0.0.0'))
```

Comment:
Is there a simple way to get the actual SAL version? That could be a separate ticket, but I suspect it's a small change, for instance if there's a text file in a known location that has the version in an easily parsed form.

Reply:
It is in ts_sal/VERSION or getSALVersion() in the API

Reply:
I don't think we want the doc build to rely on a SALPY library, since that means we have to build it first. That suggests parsing ts_sal/VERSION in this file -- or in python library code so we can have the usual `lsst.ts.sal.__version__` attribute.